src/cf-forms/src/molecules/form-fields.less

```diff
@@ -11,11 +11,19 @@
     &__checkbox,
     &__radio {
         .a-label {
+            display: -ms-inline-grid;
+            display: inline-grid;
+            // 30px is width of checkbox/radio button plus the needed padding.
+            -ms-grid-columns: unit( 30px / @base-font-size-px, em ) 1fr;
+            grid-template-columns: unit( 30px / @base-font-size-px, em ) auto;
```

Comment:
Will Autoprefixer not handle the `-ms-` prefixes?

Reply:
It should.

Reply:
Notice the values `1fr` and `auto` are different. IE has been what's holding up this PR. It absolutely mangles the grid. I've decided to just drop the columns entirely in IE and go with a fallback state where the label doesn't have the indent.